xga/utils.py

```diff
@@ -677,8 +684,9 @@ def check_telescope_choices(telescope: Union[str, List[str]]) -> List[str]:
 ciao_err = ciao_err.decode("UTF-8")
 
 if "ciaover: command not found" in ciao_err:
-        warn("No CIAO installation detected on system, "
-             "as such all functions in xga.generate.ciao will not work.", stacklevel=2)
+    split_out = ''
```

Comment:
Thanks for spotting this problem - I missed it when I did code review for that PR. Rather than defining a blank split out, please could you check if CIAO_VERSION is None further down when it gets to the CALDB version check that was failing - I'll put another comment where I mean

Reply:
This doesn't work when I try to implement it, if I don't define `split_out` the code falls over at the if statement later because `split_out` is not defined anywhere. Alternatively, setting `split_out = [en.strip(' ') for en in ciao_out.split('\n')]` also doesn't work because `ciao_out` is empty if `ciao` is not installed/initialised. Using `ciao_err` instead of `ciao_out` throws an error too because in that case split_out has less than 6 elements so the indexing in the if statement stops working.

I think we found this was happening when we wrote this which is why we defined the blank `split_out` here.